lib/services.rb

```diff
@@ -4,7 +4,11 @@ module Services
   def self.publishing_api
     @publishing_api ||= GdsApi::PublishingApiV2.new(
       Plek.new.find('publishing-api'),
-      disable_cache: true,
+      cache_ttl: 5.minutes,
```

Comment:
Wondering if we can do this for just the linkables. I'm worried that responses for `get_content_item` will also, be cached. That could cause confusion if the user sees the old version after editing a taxon.